#==================================================================================================
# From ce030b1b36e7c63a3c120417cf9b3fdd9d3bdb80 Mon Sep 17 00:00:00 2001
# From: Matthias Clasen <mclasen@redhat.com>
# Date: Sat, 13 Apr 2024 22:28:48 -0400
# Subject: [PATCH] gsk: Fix a minor type mismatch
#
# Use the same types in the declaration of gsk_standard_contour_init.
#
# Issue:         https://gitlab.gnome.org/GNOME/gtk/-/issues/6628
# Merge Request: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/7138
#==================================================================================================
--- gsk/gskcontour.c
+++ gsk/gskcontour.c
@@ -556,7 +556,7 @@ gsk_standard_contour_init (GskContour             *contour,
                            gsize                   n_points,
                            const gskpathop        *ops,
                            gsize                   n_ops,
-                           ptrdiff_t               offset);
+                           gssize                  offset);
 
 static void
 gsk_standard_contour_copy (const GskContour *contour,
